CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 - v1.14.x
 - v1.13.x
 
+### Improvements
+
+-   Automatically mark Secret data and stringData as secret. (https://github.com/pulumi/pulumi-kubernetes/pull/803).
```

Comment:
We should add clarifying docs that this secret encoding only applies to the statefile, as the secret will still be stored in plaintext or b64 in k8s and etcd by default. 

As of 1.13 there is a beta feature for [encryption at rest](https://kubernetes.io/docs/concepts/configuration/secret/#security-properties) for etcd, assuming the user can enable this feature. This would also be a good time to remind folks to lock down etcd and control plane access to admins only if possible - see [risks](https://kubernetes.io/docs/concepts/configuration/secret/#risks).

IMO we should present both links ^ in the docs.

Reply:
@metral Which docs were you thinking? The SDK docs for the Secret class?

Reply:
I was leaning towards the ones linked in my prior comment. Figured this would be a good opportunity to nudge the user to consider reading both links and applying them if possible

Reply:
Added a comment to the SDK docs for the Secret classes.